File: 3, Loops/check_number_is_prime.java
import java.util.*;

public class check_number_is_prime{
    public static void main(String args[]){
        Scanner scan = new Scanner(System.in);
        int number = scan.nextInt();
        boolean check = true;
        for(int i=3;i<number;i++){
            if((number%i)==0){
                check = false;
                break;
            }
        }
        if(check==true) System.out.println(number+" is a prime number");
        else System.out.println(number+" is not a prime number");
    }
}

File: C++ course/Some Basic Math/check_number_is_prime.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int num;
    cin>>num;
    bool check = true;

    for(int i=2;i<num;i++) if(num%i==0) check = false;

    if(check == true) cout<<"The number is prime number"<<endl;
    else cout<<"The number is not prime number"<<endl;
    return 0;
}